hackathon_site/dashboard/frontend/src/slices/event/teamDetailSlice.ts

```diff
@@ -30,6 +30,46 @@ interface RejectValue {
     message: any;
 }
 
+interface idProvidedParameters {
```

Comment:
I see what you're doing here but good news is patch requests don't require all properties in the request JSON to be there! you don't need the attended property and the request will still work, so you can remove it.

Reply:
oh cool, fixed